### PR TITLE
Adds email address format support

### PR DESCRIPTION
**Pull request by ghost**
I need to get the entire email address back from RailsValues, delegates format to the format method on `Mail` rather than meddling with `to_s` of  `RailsValues::EmaliAddress`

### CODE REVIEW

File: rails_values.gemspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name          = 'rails_values'
-  spec.version       = '1.2.0'
+  spec.version       = '1.2.1'
```
Comment (grantspeelman) [inline rails_values.gemspec:3]: it's not a bug fix, so should be a minor version increase

```suggestion
  spec.version       = '1.3.0'
```

**Review by grantspeelman — APPROVED** [review]
Nice Work 👍🏽 
one small comment on the version

**Comment by ghost** [comment]
> Nice Work 👍🏽 one small comment on the version

@grantspeelman  Ergh - sorry - auto merged, I'll fix it.

**Comment by grantspeelman** [comment]
> > Nice Work 👍🏽 one small comment on the version
> 
> @grantspeelman Ergh - sorry - auto merged, I'll fix it.

no worries

**Pull request merged**